sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeFormatter.scala

```diff
@@ -116,29 +127,32 @@ sealed trait DateFormatter {
 
 class Iso8601DateFormatter(
     pattern: String,
-    timeZone: TimeZone,
-    locale: Locale) extends DateFormatter {
+    locale: Locale) extends DateFormatter with FormatterUtils {
 
-  val dateTimeFormatter = new Iso8601DateTimeFormatter(pattern, timeZone, locale)
+  val zoneId = ZoneId.of("GMT")
```

Comment:
maybe doesn't matter, but `UTC` is more widely used when timezone doesn't matter.

Reply:
see https://www.timeanddate.com/time/gmt-utc-time.html

Reply:
Thanks for the link. I probably read the same when I thought what is better to use. What confused me is **GMT is a time zone** vs **UTC is not a time zone**

Reply:
`UTC` is a time standard, and `GMT` is `UTC+0`. So here looks like `UTC` is more proper.